### ISSUE TITLE
Text-muted contrast

### ISSUE THREAD

**Issue by brianteeman** [j2commerce/j2commerce#495]
the text-muted class in atum fails accessibility in light mode. This is a joomla bug but it heavily impacts j2commerce. Core doesnt really use the class but j2commerce does.

rarther than trying to change the css in j2commerce I recommend that you just change all uses of text-muted to small

**Comment by newlinewebdesign** [comment]
Agreed. I was never a fan of the text-muted display

**Comment by brianteeman** [comment]
if its easier you can just change the class to small but it would be more semantic correct to change the div to < small > etc

**Comment by newlinewebdesign** [comment]
i worry that changing the div to small would also change it from a block element to inline element - i will see how many times text-muted is used so I can see what we are really looking at...

**Comment by brianteeman** [comment]
Good point - I forgot small is an inline element

**Comment by newlinewebdesign** [comment]
Fix submitted in PR #518

**Comment by brianteeman** [comment]
The fix doesnt work as other css has priority - and even if it did work you need to address the opacity as this turns your valid contrast into a failing contrast. Also please test in dark mode. Oh and my suggestion to just change everything to small is also wrong as there are text-muted p as shown in the second image

<img width="716" height="688" alt="Image" src="https://github.com/user-attachments/assets/01ce9ede-3374-41c4-a193-ee29c5c172d6" />

<img width="1015" height="592" alt="Image" src="https://github.com/user-attachments/assets/635c0b72-5cba-4c8e-91ba-d05ff52cb89e" />

**Comment by newlinewebdesign** [comment]
good catch - just checking to make sure you are paying attention ;)  working on it now
